en/docs/learn/user-account-suspension.md

```diff
@@ -118,7 +118,11 @@ task that checks for idle accounts is common to all tenants.
-    | Lock Timeout Increment Factor | This specifies the increment factor of the lock timeout. If the lock timeout increment factor is set to 2, then the timeout increments by 2 to the power of the number of failed login attempts. [Timeout increment = (Lock Timeout Increment Factor)<sup>(Number of Failed Attempts)</sup>] (If the number of failed login attempts is 3, timeout increment is 2<sup>3</sup>=8)                                                                                                                                                                                                                                          | 2           |
+    | Account Lock Enabled | This specifies the account locking for failed logins.                                                                                                                                                                                                                                         |            |
+    | Maximum Failed Login Attempts | This specifies the number of failed attempts allows without locking the account.                                                                                                                                                                                                                                      | 5           |
+    | Lock Unlock time | This specifies the account locked time span in minutes.                                                                                                                                                                                                                                          | 5           |
+    | Lock Timeout Increment Factor | This specifies the increment factor of the lock timeout. If the lock timeout increment factor is set to 2, then the timeout increments by 2 to the power of the number of failed login attempts. [Timeout increment = (Lock Timeout Increment Factor)<sup>(Number of Failed Attempts)</sup>] (If the number of failed login attempts is 3, timeout increment is 2<sup>3</sup>=8)                                                                                                                                                                                                                                           |  2          |
+    | Enable Notification Internally Management | The feild is set false if the client application handles notification sending.                                                                                                                                                                                                                                           |            |
```

Comment:
This field should be set to 'false'...

Reply:
Fixed https://github.com/wso2/docs-is/pull/1409/commits/fcb53cb5030e00b3a00d9054f1bb75f1e310757e

Reply:
The commit still shows the same line..

Please note that 'feild' should be corrected to 'field'

Reply:
Since the above lines are outdated please check the updated user-account-suspension.md in https://github.com/wso2/docs-is/commit/fcb53cb5030e00b3a00d9054f1bb75f1e310757e commit.

The spelling mistake was corrected in https://github.com/wso2/docs-is/pull/1409/commits/377f612e5f8ec34fd29c509b6f760d3b4c4a7c55 commit.

Reply:
Yes. It has been updated now with commit 377f612